app/src/main/java/com/hardsoftstudio/rxflux/sample/ui/mainactivity/MainActivity.java

```diff
@@ -1,4 +1,4 @@
-package com.hardsoftstudio.rxflux.sample;
+package com.hardsoftstudio.rxflux.sample.ui.mainactivity;
```

Comment:
why do you create a package just for this activity? I am agree on creating the ui package but not the mainactivity